### DISCUSSION TITLE
How can I get the modify time

### DISCUSSION THREAD

**Discussion by jdysya** [FreshRSS/FreshRSS#7004] [Q&A]
I checked the table structure of the database and found that there is no data modification time, I can't get the exact time of reading or bookmarking the article.

But I found the response of the [inoreader api](https://www.inoreader.com/developers/stream-contents) have `updated` field.

<img width="778" alt="image" src="https://github.com/user-attachments/assets/02392218-2a40-428f-be1b-824c80fc6c26">

I would like to know why this project does not provide a updated time. Would you consider adding this field in the future, right now I would like to count the reads of the day and get confused when I use the greader api.

**Comment by Alkarex** [comment]
Hello,
In the GReader API, the `updated` field indicates when the content has last been updated. It does not tell when the user has performed an action such as marking as read/unread or as favourite.
At the moment, we do not keep track of when user actions took place, but that could be an interesting feature to add.

See this related discussion - help welcome https://github.com/FreshRSS/FreshRSS/discussions/6878
